web/channels/game_channel.ex

```diff
@@ -24,9 +21,18 @@ defmodule Pirates.GameChannel do
     {:noreply, socket}
   end
 
-  def handle_info({:state_tick, state}, socket) do
-    # filter out the client's state from the map
-    other_state = Enum.reject state, fn %{id: id} ->
+  # def handle_info({:state_tick, state}, socket) do
+  #   # filter out the client's state from the map
+  #   other_state = Enum.reject state, fn %{id: id} ->
+  #     socket.assigns.id == id
+  #   end
+  #   broadcast_from socket, "state_tick", %{state: other_state}
+  #   {:noreply, socket}
+  # end
+
+  intercept ["state_tick"]
+  def handle_out("state_tick", %{state: states}, socket) do
```

Comment:
this isn't needed if you use https://hexdocs.pm/phoenix/Phoenix.Channel.html#broadcast_from/3 as I commented earlier

Reply:
See above comment.